internalserver/example_test.go

```diff
@@ -0,0 +1,79 @@
+package internalserver
```

Comment:
I still think just `server` would be enough.

Reply:
I disagree, as this is an internalserver specifically. It's not some server that you would expose to the world. I thought about calling it `internal` but that's obviously a special package name in Go.

Reply:
Naming :shrug: as long as it's clear when a consumer package uses it: `internalserver.NewHandler(...)`